examples/cockroachdb/README.md

```diff
@@ -0,0 +1,248 @@
+# CockroachDB
+
+[CockroachDB](https://www.cockroachlabs.com/) is one of the database servers based on SQL.
+
+## Introduction
+
+This chart bootstraps a three node Cockroachdb deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.16+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.80.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
+## Installing the Chart
+
+To install the CockroachDB database using the `cockroachdb` chart with the release name `cockroachdb-release`:
+
+```bash
+# Add cockroachdb in your local chart repository
+$ helm repo add cockroachdb https://charts.cockroachdb.com/
+
+# Update your local chart repository
+$ helm repo update
+
+# Install the cockroachdb in cockroachdb namespace
+$ helm install cockroachdb-release --namespace cockroachdb --create-namespace cockroachdb/cockroachdb
+
+# Install the secure cockroachdb client
+$ curl -OOOOOOOOO https://raw.githubusercontent.com/cockroachdb/helm-charts/master/examples/client-secure.yaml
+
+# Edit Fields in client-secure.yaml
+spec.serviceAccountName: cockroachdb-release
+spec.image: cockroachdb/cockroach:<VERSION>
+spec.volumes[0].project.sources[0].secret.name: cockroachdb-release-client-secret
+
+$ kubectl create -f ./client-secure.yaml -n cockroachdb
+
+
+```
+> **Note** : 
+cockroachdb/cockroachdb version in spec.image as mentioned in following chart [values.yaml](https://github.com/cockroachdb/helm-charts/blob/master/cockroachdb/values.yaml)
+
+The command deploys a CockroachDB instance in the `cockroachdb` namespace.
+
+> **Tip**: List all releases using `helm list --all-namespaces`, using Helm Version 3.
+
+## Integrating with Kanister
+
+If you have deployed CockroachDB application with name other than `cockroachdb-release` and namespace other than `cockroachdb`, you need to modify the commands(backup, restore and delete) used below to use the correct release name and namespace
+
+### Create Secret and Configmaps
+Create Secret and Configmap if not created already and Edit the values accordingly
+
+```bash
+$ kubectl create -f ./secret.yaml
+$ kubectl create -f ./configmap.yaml
+```
+
```

Comment:
I dont see a step to create kanister profile, since we are using profile in blueprint we should add that step in the readme